jakarta.ls/src/main/java/io/microshed/jakartals/JakartaTextDocumentService.java

```diff
@@ -93,6 +93,21 @@ private void triggerValidationForAll() {
 		List<String> allDocs = documents.all().stream().map(doc -> doc.getUri()).collect(Collectors.toList());
 		triggerValidationFor(allDocs);
 	}
+	
+	
+	@Override
+	public CompletableFuture<Hover> hover(HoverParams params) {
+		// TODO give hover message to the server
+		LOGGER.info("Getting the hover going at server");
```

Comment:
Remove this log message or change it to something like `received textDocument/hover request`